### ISSUE TITLE
Transaction notifications after player becomes online

### ISSUE THREAD

**Issue by Bomfunk** [EpicEricEE/ShopChest#47]
Hello!

I might be missing something, but I haven't found a way of configuring notifications for players that were offline when someone bought/sold something from/to them. It seems that there is no way for them to be informed other than being online (I haven't tested that yet - they do get notified, right?)

So, am I missing something or could this be considered as a feature request?

Thanks in advance.

**Comment by EpicEricEE** [comment]
That's not possible yet, but wouldn't the vendor's chatbox be spammed with messages after join when multiple players buy or sell a lot to the shop?

**Comment by Volander** [comment]
Perhaps just with a net sum of what profit you have earned while being gone.

**Comment by Bomfunk** [comment]
It does not have to be full log of all transactions, but it could be at least useful to see some sort of summary.
I set up a shop at my server and users buy and sell stuff there, but I am not sure how to check what exactly was happening, but I do notice my balance changing back and forth...

I noticed that all transactions are actually logged in the database. Maybe a command like "/shop log", showing latest transactions made with ability to turn pages ("/shop log 2", etc.), could be useful? Of course, it should only show transactions for the shops user owns.

Thank you!
